cmd/explaintest/r/partition_pruning.result

```diff
@@ -4273,3 +4273,15 @@ Union_9	750.00	root
   └─Selection_17	250.00	cop	ge(test.t1.a, 10), le(test.t1.a, 43)
     └─TableScan_16	10000.00	cop	table:t1, partition:p2, range:[-inf,+inf], keep order:false, stats:pseudo
 drop table t0, t1;
+drop table if exists t;
+create table t(a timestamp) partition by range(unix_timestamp(a)) (partition p0 values less than(unix_timestamp('2019-02-16 14:20:00')), partition p1 values less than (maxvalue));
+insert into t values('2019-02-16 14:19:59'), ('2019-02-16 14:20:01');
+explain select * from t where a between time'14:19:00' and timestamp'2019-02-16 14:21:00';
+id	count	task	operator info
+Union_8	500.00	root	
+├─TableReader_11	250.00	root	data:Selection_10
+│ └─Selection_10	250.00	cop	ge(test.t.a, 2019-02-16 14:19:00), le(test.t.a, 2019-02-16 14:21:00)
```

Comment:
Why it is `2019-02-16 14:19:00`? Does it depend on the date when the test run?

Reply:
Yes, because time'14:19:00' is cast to timestamp and it's 'today' + time. I'll change it to `timestamp'2019-02-16 14:19:00'` to make the test stable.